Conceptpower+Spring/src/main/java/edu/asu/conceptpower/servlet/web/ConceptEditController.java

```diff
@@ -258,6 +267,47 @@ public ModelAndView confirmEdit(@PathVariable("id") String id, HttpServletReques
         return new ResponseEntity<String>(buildJSON(synonymList, false, true), HttpStatus.OK);
     }
 
+    @RequestMapping(method = RequestMethod.GET, value = "conceptEdit/conceptSearch")
+    public @ResponseBody ResponseEntity<Object> searchConcept(@RequestParam("concept") String concept,
```

Comment:
@jdamerow  Yes I was thinking the same. While creating the concept we are returning the model object but here we are returning the response entity. May be I can combine the logic of searching a concept. While create a new concept we also get the concept list and concept types details, Do we need the same details here too?